<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestForm</class>
 <widget class="QWidget" name="TestForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="statisticHorizontalLayout">
       <item>
        <spacer name="statisticHorizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="currentNumberLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="slashLabel">
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="questionsSizeLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="buttonHorizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="menuPushButton">
         <property name="text">
          <string>Menu</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="nextPushButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>menuPushButton</sender>
   <signal>clicked()</signal>
   <receiver>TestForm</receiver>
   <slot>BackToMenu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>-9</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextPushButton</sender>
   <signal>clicked()</signal>
   <receiver>TestForm</receiver>
   <slot>NextQuestion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>627</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>748</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextPushButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>nextPushButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>627</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>627</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextPushButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>TestForm</receiver>
   <slot>CheckQuestionState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>627</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>733</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextPushButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>TestForm</receiver>
   <slot>CheckQuestionState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>627</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>694</x>
     <y>493</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>BackToMenu()</slot>
  <slot>NextQuestion()</slot>
  <slot>CheckQuestionState()</slot>
 </slots>
</ui>
